2021-05-05 16:34:48,400 | 3484 | proxy.proxy, start_server, 199 | INFO | Server on ('127.0.0.1', 63333)
2021-05-05 16:34:48,401 | 3484 | proxy.proxy, start_server, 202 | INFO | Server serve forever ('127.0.0.1', 63333)
2021-05-05 16:34:55,737 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33290) is connected
2021-05-05 16:34:55,747 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33294) is connected
2021-05-05 16:34:55,751 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33296) is connected
2021-05-05 16:34:55,753 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:34:55,754 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00F\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:34:55,755 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00F\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:34:55,755 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:34:55,756 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:34:55,759 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:34:55,759 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08#\x14i\xc1'
2021-05-05 16:34:55,760 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08#\x14i\xc1'
2021-05-05 16:34:55,760 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:34:55,761 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:34:55,761 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00G\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:34:55,762 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00G\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:34:55,762 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:34:55,763 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:34:55,763 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00H\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:34:55,763 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00H\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:34:55,764 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:34:55,764 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:34:55,766 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:34:55,766 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08)Z\xa9&'
2021-05-05 16:34:55,767 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08)Z\xa9&'
2021-05-05 16:34:55,767 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:34:55,768 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:34:55,768 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:34:55,768 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x0890\xb6\xf3'
2021-05-05 16:34:55,769 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x0890\xb6\xf3'
2021-05-05 16:34:55,771 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:03,757 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:03,758 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:03,759 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:03,760 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:03,761 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:03,762 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:03,762 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:03,763 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:03,765 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:11,769 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:35:11,770 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-11' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c3f760>()]>>
2021-05-05 16:35:11,771 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:35:11,772 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-12' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c3f8b0>()]>>
2021-05-05 16:35:11,772 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:35:11,773 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-13' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c3fe80>()]>>
2021-05-05 16:35:11,773 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33306) is connected
2021-05-05 16:35:11,784 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33308) is connected
2021-05-05 16:35:11,793 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33310) is connected
2021-05-05 16:35:11,801 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:11,808 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00J\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:11,811 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00J\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:11,815 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:11,816 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:11,819 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00K\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:11,824 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00K\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:11,825 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:11,826 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:11,826 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00L\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:11,827 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00L\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:11,828 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:11,829 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:11,830 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:11,831 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b"\x04\x00\x00\x08\xf2\xc1'."
2021-05-05 16:35:11,832 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b"\x04\x00\x00\x08\xf2\xc1'."
2021-05-05 16:35:11,832 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:11,836 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:11,838 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:11,839 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:11,841 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:11,846 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08 T\x8d\xdd'
2021-05-05 16:35:11,852 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08 T\x8d\xdd'
2021-05-05 16:35:11,860 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:11,867 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\x9c\x90A\x9f'
2021-05-05 16:35:11,871 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\x9c\x90A\x9f'
2021-05-05 16:35:11,871 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:19,780 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:19,781 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:19,781 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:19,782 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:19,783 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:19,783 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:19,784 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:19,784 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:19,784 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:27,798 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33320) is connected
2021-05-05 16:35:27,803 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33322) is connected
2021-05-05 16:35:27,808 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33324) is connected
2021-05-05 16:35:27,823 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:27,824 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00M\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:27,824 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00M\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:27,825 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:27,826 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:27,826 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00N\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:27,827 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00N\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:27,827 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:27,828 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:27,829 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00O\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:27,829 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00O\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:27,835 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:27,835 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:27,836 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:27,836 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08=/\xab\x02'
2021-05-05 16:35:27,838 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08=/\xab\x02'
2021-05-05 16:35:27,839 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:27,840 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:27,840 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:27,841 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\x8a\xd1\xd8\x84'
2021-05-05 16:35:27,841 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\x8a\xd1\xd8\x84'
2021-05-05 16:35:27,842 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:27,842 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:27,847 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:27,848 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08d\x05\xc1\x10'
2021-05-05 16:35:27,849 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08d\x05\xc1\x10'
2021-05-05 16:35:27,850 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:35,862 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:35,863 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:35,864 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:35,866 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:35,868 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:35,869 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:35,871 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:35,871 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:35,872 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:43,877 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33334) is connected
2021-05-05 16:35:43,882 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33336) is connected
2021-05-05 16:35:43,887 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33338) is connected
2021-05-05 16:35:43,892 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:43,899 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00P\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:43,899 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00P\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:43,900 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:43,900 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:43,901 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00Q\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:43,902 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00Q\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:43,902 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:43,903 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:43,904 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00R\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:43,905 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00R\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:43,906 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:43,906 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:43,907 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:43,911 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\x81H\xe1\xaa'
2021-05-05 16:35:43,911 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\x81H\xe1\xaa'
2021-05-05 16:35:43,911 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:43,912 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:43,912 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:43,912 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\xf7\x893\x96'
2021-05-05 16:35:43,913 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\xf7\x893\x96'
2021-05-05 16:35:43,913 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:43,914 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:43,915 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:43,921 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\xf0E6j'
2021-05-05 16:35:43,929 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\xf0E6j'
2021-05-05 16:35:43,959 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:51,890 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:51,890 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:51,891 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:51,891 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:51,892 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:51,892 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:51,893 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:35:51,893 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:35:51,894 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:35:59,904 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33348) is connected
2021-05-05 16:35:59,909 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:59,911 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:59,914 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:59,919 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\x8c5T\xaf'
2021-05-05 16:35:59,919 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\x8c5T\xaf'
2021-05-05 16:35:59,920 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:59,927 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00S\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:59,930 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00S\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:59,930 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:59,932 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33352) is connected
2021-05-05 16:35:59,939 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33354) is connected
2021-05-05 16:35:59,945 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:59,949 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00T\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:59,950 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00T\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:59,950 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:59,951 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:59,956 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00U\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:59,963 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00U\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:35:59,971 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:35:59,974 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:59,975 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:59,977 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\x91\xc1\xcb\xc2'
2021-05-05 16:35:59,982 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\x91\xc1\xcb\xc2'
2021-05-05 16:35:59,984 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:59,987 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:35:59,990 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:35:59,991 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\xca\xfc\x91I'
2021-05-05 16:35:59,992 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\xca\xfc\x91I'
2021-05-05 16:35:59,997 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:07,916 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:07,917 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:07,917 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:36:07,919 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:07,919 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:07,920 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:36:07,920 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:07,921 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:07,921 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:36:15,928 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33362) is connected
2021-05-05 16:36:15,932 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33364) is connected
2021-05-05 16:36:15,935 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33366) is connected
2021-05-05 16:36:15,939 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:15,943 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00V\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:15,944 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00V\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:15,944 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:15,945 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:15,945 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00W\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:15,945 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00W\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:15,946 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:15,946 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:15,947 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00X\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:15,947 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00X\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:15,954 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:15,955 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:36:15,956 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:15,957 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b"\x04\x00\x00\x08\x96\xba'\xc1"
2021-05-05 16:36:15,957 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b"\x04\x00\x00\x08\x96\xba'\xc1"
2021-05-05 16:36:15,957 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:15,958 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:36:15,958 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:15,958 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08S\x8b\x0c\x9d'
2021-05-05 16:36:15,959 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08S\x8b\x0c\x9d'
2021-05-05 16:36:15,959 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:15,959 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:36:15,960 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:15,960 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\xb2s\x87:'
2021-05-05 16:36:15,961 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\xb2s\x87:'
2021-05-05 16:36:15,961 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:23,941 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:23,944 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:23,945 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:36:23,946 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:23,946 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:23,947 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:36:23,948 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:23,948 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:23,949 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,953 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,954 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-20' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c3f0a0>()]>>
2021-05-05 16:36:31,955 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,956 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-21' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c3fac0>()]>>
2021-05-05 16:36:31,956 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,957 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-22' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c3f880>()]>>
2021-05-05 16:36:31,957 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,958 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-29' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c5a640>()]>>
2021-05-05 16:36:31,959 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,959 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-30' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c5a4c0>()]>>
2021-05-05 16:36:31,961 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,962 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-31' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c5a310>()]>>
2021-05-05 16:36:31,964 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,970 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-38' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c5ae20>()]>>
2021-05-05 16:36:31,971 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,972 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-39' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c5aca0>()]>>
2021-05-05 16:36:31,972 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,973 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-40' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c5aaf0>()]>>
2021-05-05 16:36:31,974 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,975 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-43' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44be50a0>()]>>
2021-05-05 16:36:31,977 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,978 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-48' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44be5640>()]>>
2021-05-05 16:36:31,979 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,989 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-49' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44be5490>()]>>
2021-05-05 16:36:31,989 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,990 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-56' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44be5e20>()]>>
2021-05-05 16:36:31,992 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,993 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-57' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44be5ca0>()]>>
2021-05-05 16:36:31,994 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:36:31,997 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-58' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44be5af0>()]>>
2021-05-05 16:36:32,009 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33376) is connected
2021-05-05 16:36:32,022 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33378) is connected
2021-05-05 16:36:32,026 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33380) is connected
2021-05-05 16:36:32,043 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:32,048 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00Y\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:32,052 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00Y\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:32,063 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:32,064 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:32,064 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00Z\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:32,065 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00Z\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:32,066 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:32,067 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:32,067 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00[\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:32,070 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00[\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:32,073 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:32,073 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:36:32,074 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:32,074 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\x9c\xf9\x8e\xf8'
2021-05-05 16:36:32,075 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\x9c\xf9\x8e\xf8'
2021-05-05 16:36:32,076 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:32,076 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:36:32,079 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:32,080 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08=\xfe\xda\xa5'
2021-05-05 16:36:32,081 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08=\xfe\xda\xa5'
2021-05-05 16:36:32,081 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:32,082 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:36:32,082 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:32,083 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08e\t;\xdf'
2021-05-05 16:36:32,083 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08e\t;\xdf'
2021-05-05 16:36:32,087 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:39,965 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:39,966 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:39,966 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:36:39,967 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:39,968 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:39,969 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:36:39,970 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:39,970 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:39,971 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:36:47,979 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33390) is connected
2021-05-05 16:36:47,982 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33392) is connected
2021-05-05 16:36:47,983 | 3484 | proxy.proxy, handle_switch, 216 | INFO | Client ('127.0.0.1', 33394) is connected
2021-05-05 16:36:47,990 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:48,001 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00\\\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:48,002 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00\\\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:48,002 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:48,003 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:48,003 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00]\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:48,004 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00]\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:48,005 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:48,007 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:48,008 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b'\x04\x00\x00\x10\x00\x00\x00^\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:48,009 | 3484 | proxy.proxy, send_to_controller, 72 | WARNING | Failed to send to controller : b'\x04\x00\x00\x10\x00\x00\x00^\x00\x01\x00\x08\x00\x00\x00\x10'
2021-05-05 16:36:48,012 | 3484 | proxy.proxy, handle_switch, 227 | DEBUG | waiting data ......
2021-05-05 16:36:48,014 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:36:48,015 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:48,015 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\x08;\x80\x11'
2021-05-05 16:36:48,016 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\x08;\x80\x11'
2021-05-05 16:36:48,016 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:48,017 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:36:48,021 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:48,028 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08t\xe4\x1e\x0c'
2021-05-05 16:36:48,028 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08t\xe4\x1e\x0c'
2021-05-05 16:36:48,028 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:48,029 | 3484 | proxy.proxy, handle_controller, 329 | INFO | connected Controller ('127.0.0.1', 6633)
2021-05-05 16:36:48,029 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:48,030 | 3484 | proxy.proxy, handle_controller, 336 | DEBUG | read b'\x04\x00\x00\x08\xe7\xab\x82\xa9'
2021-05-05 16:36:48,031 | 3484 | proxy.proxy, send_to_switch, 90 | WARNING | Failed to send to switch : b'\x04\x00\x00\x08\xe7\xab\x82\xa9'
2021-05-05 16:36:48,031 | 3484 | proxy.proxy, handle_controller, 334 | DEBUG | waiting data ......
2021-05-05 16:36:55,991 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:55,992 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:55,993 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:36:55,994 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:55,994 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:55,995 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:36:55,995 | 3484 | proxy.proxy, handle_switch, 229 | DEBUG | read b''
2021-05-05 16:36:55,996 | 3484 | proxy.proxy, handle_switch, 231 | DEBUG | no data from switch read
2021-05-05 16:36:55,996 | 3484 | proxy.proxy, handle_switch, 246 | INFO | Close the connection and do exit processing
2021-05-05 16:37:00,369 | 3484 | root, <module>, 51 | INFO | keyboardInterrupt : 
2021-05-05 16:37:00,380 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:37:00,387 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-65' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44be5c40>()]>>
2021-05-05 16:37:00,388 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:37:00,389 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-66' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44be58e0>()]>>
2021-05-05 16:37:00,389 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:37:00,394 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-67' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44be59a0>()]>>
2021-05-05 16:37:00,395 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:37:00,403 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-74' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c5ae20>()]>>
2021-05-05 16:37:00,404 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:37:00,406 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-75' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c5aca0>()]>>
2021-05-05 16:37:00,412 | 3484 | proxy.proxy, handle_controller, 349 | INFO | Close the connection and do exit processing
2021-05-05 16:37:00,413 | 3484 | asyncio, default_exception_handler, 1707 | ERROR | Task was destroyed but it is pending!
task: <Task pending name='Task-76' coro=<ControllerHandler.handle_controller() done, defined at /home/shu/Network/ofcapture/proxy/proxy.py:317> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fba44c5aaf0>()]>>
